create_db.js

```diff
@@ -0,0 +1,18 @@
+import pg from 'pg';
+import config from './config/config';
+
+const conStringPri = `postgres://${config.postgres.user}:${config.postgres.passwd}@${config.postgres.host}:${config.postgres.port}/postgres`;
+
+pg.connect(conStringPri, (err, client, done) => { // eslint-disable-line no-unused-vars
+  // create the db and ignore any errors, for example if it already exists.
+    client.query(`CREATE DATABASE ${config.postgres.db}`, (err1) => { // eslint-disable-line no-unused-vars
+    // If in test mode and a database exists don't let it get created
+        if (err1) {
+            console.log('Database Creation Failed');
```

Comment:
Can we do a better error message. Something like "Database creation failed. This can happen if the database specified already exists."